test/built-ins/Proxy/get/trap-is-undefined-receiver.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.8
```

Comment:
In addition to es6id could you please add esid: sec-proxy-object-internal-methods-and-internal-slots-get-p-receiver